app/src/main/java/org/sil/gatherwords/SettingsActivityCustom.java

```diff
@@ -0,0 +1,21 @@
+package org.sil.gatherwords;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class SettingsActivityCustom extends AppCompatActivity {
```

Comment:
Could you rename this?  `XXCustom` gives very little idea of what this class is relevant to.